src/js/fable-compiler/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fable-compiler",
-  "version": "2.0.0-alpha-001",
+  "version": "2.0.0-alpha-002",
```

Comment:
It's better if you don't change the version in package.json because the "PublishPackages" build target checks if RELEASE_NOTES and package.json are out of sync and in that case, it publishes the package.

Reply:
@alfonsogarciacaro Thanks, let me know once the CI can be used to trigger the publishing, as I want to get the -002 version published before people start using it.